account_invoice_supplier_ref_unique/tests/test_account_invoice_supplier_ref_unique.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2016 Acsone
+# Copyright 2020 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields
+from odoo.exceptions import ValidationError
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestAccountInvoiceSupplierRefUnique(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        # ENVIRONMENTS
+        cls.account_account = cls.env["account.account"]
+        cls.account_move = cls.env["account.move"].with_context(
+            **{"tracking_disable": True}
+        )
+
+        # INSTANCES
+        cls.partner = cls.env.ref("base.res_partner_2")
```

Comment:
Can you create a partner instead of relying on demo data?

Reply:
Updated, Thanks